docs/src/modules/components/Ad.js

```diff
@@ -64,49 +64,56 @@ const inHouseAds = [
   {
     name: 'scaffoldhub',
     link: 'https://scaffoldhub.io/?partner=1',
-    img: '/static/in-house/scaffoldhub.png',
+    img: '/static/images/in-house/scaffoldhub.png',
```

Comment:
I don't think that the image prefix adds anything. I think that the flatter, the better. In our case, we group per concern.

```suggestion
    img: '/static/in-house/scaffoldhub.png',
```

Reply:
This is flatter - it's helping flatten images under images. At the moment they're mixed between images and the parent folder. We could go the other way around, and get rid of the images folder, but that's a bigger job.

Reply:
Yeah, I think that `images` should be renamed `components`

Reply:
Logic? These are images.